### ISSUE TITLE
Check and update exercise setups names

### ISSUE THREAD

**Issue by fguisso** [globocom/secDevLabs#587]
<a href="https://opensource.globo.com/hacktoberfest/">
<img width="804" alt="image" src="https://user-images.githubusercontent.com/5755568/193584790-676be6b1-73be-4137-94ae-ddd649c44432.png"></a>

## Motivation

SecDevLab provides exercises based on OWASP Top 10 since 2021 the vulnerable list has been updated, but some details inside the setup of the exercise don't be updated.

## It would be great if

All exercise setups correspond to correct exercise names.

## What we expect

Check and update all exercise setups to reflect the correct name.

![image](https://user-images.githubusercontent.com/5755568/193589860-ef32e70d-2c05-4645-8d5f-da5ef25f1ee5.png)

For example, for exercise A1 - Broken Authentication (eCommerce-API) inside `docker-compose.yml` the container name doesn't correspond to the correct numeration, update `a5` to `a1`: [docker-compose.yml#5](https://github.com/globocom/secDevLabs/blob/904a2fb969cb9fb0db30b50709072da8f9d57002/owasp-top10-2021-apps/a1/ecommerce-api/deployments/docker-compose.yml#L5)

**Comment by tombenevides** [comment]
Hi. Can I work on this issue?

**Comment by fguisso** [comment]
YES @tombenevides! You can work on that and link to a draft PR or if you do not show activity, probably anyone will get this issue too.
